communication-service/src/main/java/com/example/communicationservice/controller/ChatFileController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.communicationservice.controller;
+
+import com.example.communicationservice.controller.api.ChatFileControllerApi;
+import com.example.communicationservice.controller.dto.request.ChatFileUploadUrlGenerateRequest;
+import com.example.communicationservice.controller.dto.response.ChatFileUploadUrlGenerateResponse;
+import com.example.communicationservice.service.ChatFileService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.hexagon.core.dto.ResponseDto;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/chatrooms/{room-id}/files")
+public class ChatFileController implements ChatFileControllerApi {
+
+    private final ChatFileService chatFileService;
+
+    // 파일 업로드 URL 생성 API
+    @PostMapping("/upload-url")
```

Comment:
Path variable이 URL에 없습니다.
```suggestion
    @PostMapping("/upload-url/{room-id}")
```

Reply:
클래스 레벨에 적용해둔 상태입니다!
계층적 구조를 나타내고 싶어서 채팅방 -> 파일 -> 업로드 url 순으로 명시했습니다.

```java
@RequestMapping("/api/chatrooms/{room-id}/files")
```